commons/src/main/java/io/hyscale/commons/commands/CommandExecutor.java

```diff
@@ -200,4 +209,18 @@ private static OutputStream getOutputStream(File outputFile) throws FileNotFound
         return new StringOutputStream();
     }
     
+    
+	public static void executeExec(List<String> commands) {
```

Comment:
Instead of eating exception we need to throw HyscaleException.
For commands should we have the argument as List or String?
Add null checks where required.
Also should we return exit code to know if command was successful or not?
Also what were the values of exit code if someone stops the command?

Reply:
method name can be better like executeWithParentIO , should be relevant to class contract